src/data/sign.lean

```diff
@@ -174,6 +174,14 @@ end cast
   map_one'  := rfl,
   map_mul'  := λ x y, by cases x; cases y; simp }
 
+lemma range_eq {α} (f : sign_type → α) : set.range f = {f zero, f neg, f pos} :=
+begin
+  classical,
+  simp only [← finset.coe_singleton, ← finset.image_singleton,
+    ← fintype.coe_image_univ, finset.coe_image, ← set.image_insert_eq],
+  refl,
```

Comment:
Does using `simpa` here work without `refl`?

Reply:
It does. (The proof was suggested by Junyan Xu, BTW.) I'll make the simplification.